public_html/includes/nav.php

```diff
@@ -39,6 +39,7 @@
             <li><a href="{local var="siteRoot"}admin/projects/"><i class="fa fa-angle-double-right"></i> Projects</a></li>
             <li><a href="{local var="siteRoot"}admin/readonly/"><i class="fa fa-angle-double-right"></i> Read Only</a></li>
```

Comment:
all of the i tags need to be changed to spans.  <i> is inaccessible use span.  This was probably done before you and you were mimicking what was there, but span is the appropriate usage.